frontend/src/components/Layout.js

```diff
@@ -1,76 +1,45 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
+import {
+  Container,
+  Divider,
+  Dropdown,
+  Grid,
+  Header,
+  Image,
```

Comment:
Should we import all of these now?  Not a big deal if so but not using all of them.